03-specifying-sample-designs.Rmd

```diff
@@ -659,22 +659,22 @@ des_acs <- pums_in %>%
     scale = 4 / 80
   )
 
-des_acs
+des_acs 
 
-summary(des_acs)
+summary(des_acs) 
 ```
 
-When printing the design object or looking at the summary, the replicate weight type is re-iterated as `Unstratified cluster jacknife (JK1) with 80 replicates and MSE variances`, and the variables are included. No weight or probability summary is included, as was done in some other design objects.
+When printing the design object or looking at the summary, the replicate weight type is re-iterated as `Unstratified cluster jacknife (JK1) with 80 replicates and MSE variances`, and the variables are included. No weight or probability summary is included.
 
-### Bootstrap method
+### Bootstrap Method
 
-In bootstrap resampling, replicates are created by selecting random samples of the PSUs with replacement. If there are $M$ PSUs in the sample, then each replicate will be created by selecting a random sample of $M$ PSUs with replacement. Each replicate is created independently, and the weights for each replicate are adjusted to reflect the population, generally using the same method as how the analysis weight was adjusted. 
+In bootstrap resampling, replicates are created by selecting random samples of the PSUs with replacement. If there are $M$ PSUs in the sample, then each replicate will be created by selecting a random sample of $M$ PSUs with replacement. Each replicate is created independently, and the weights for each replicate are adjusted to reflect the population, generally using the same method as how the analysis weight was adjusted.  
```

Comment:
"PSUs with replacement" should we just call this SRSWR as we have specified that above?